src/register/messages.jsx

```diff
@@ -126,6 +126,11 @@ const messages = defineMessages({
     defaultMessage: 'Enter a valid name',
     description: 'Validation message that appears when fullname contain URL',
   },
+  'name.validation.length.message': {
+    id: 'name.validation.message',
+    defaultMessage: 'Full name can not be longer than 255 symbols',
```

Comment:
Hi @DmytroAlipov, I coordinated with the product. Product suggestion is to use `Full name cannot be longer than 255 characters` Message.

Reply:
@awais-ansari ✅

Reply:
@awais-ansari I don't understand why the codecov failed